src/plugins/template/tests/functional/skip_tests_config.cpp

```diff
@@ -134,5 +134,12 @@ std::vector<std::string> disabledTestPatterns() {
     retVector.emplace_back(R"(.*ReferenceTopKTest.*topk_max_sort_none)");
     retVector.emplace_back(R"(.*ReferenceTopKTest.*topk_min_sort_none)");
 #endif
+
+#if defined(__aarch64__) || defined(__arm__)
```

Comment:
OPENVINO_ARCH_ARM || OPENVINO_ARCH_ARM64
defined in openvino/core/visibility.hpp

Reply:
By some reason this define doesn't work locally for me.
Do you want to check OPENVINO_ARCH_ARM || OPENVINO_ARCH_ARM64 on CI?